app/models/fee.rb

```diff
@@ -100,6 +100,23 @@ def group_name
     charge&.group_properties&.find_by(group:)&.invoice_display_name || group&.name
   end
 
+  def grouped_by_display
+    return '' unless charge?
+    return '' if charge.properties['grouped_by'].blank?
+
+    ' • ' + grouped_by.values.join(' • ')
```

Comment:
Style/StringConcatenation: Prefer string interpolation to string concatenation. (https://rubystyle.guide#string-interpolation)